cli/utils.js

```diff
@@ -0,0 +1,206 @@
+            where: {
+                domain,
+                username
+            }
+        })
+        if (!d) return false
+        else return true
+    },
+    async checkVerification(domain, username) {
+        if (isAdmin) return true // If sudo, skip.
+        try {
+            const records = await dns.resolveTxt(domain);
+
+            for (const record of records) {
+                for (const entry of record) {
+                    if (entry.includes('domain-verification=' + username)) {
+                        return true;
+                    }
+                }
+            }
+
+            return false;
+        } catch (err) {
+            console.error('Error:', err);
+            return false;
+        }
+    },
+    async reloadCaddy() {
+        const domains = await prisma.domain.findMany({
+            where: {
+
+            }
+        })
+        var caddy = {
+            apps: {
+                http: {
+                    servers: {
+                        srv0: {
+                            listen: [":443", ":80"],
+                            routes: [], // normal routes
+                            errors: {
+                                routes: [] // error routing
+                            }
+                        }
+                    }
+                },
+                tls: {
+                    automation: {
+                        policies: []
+                    }
+                }
+            }
+        }
+        // dn42 certification support
+
+        const dn42Domains = domains.filter(domain => domain.domain.endsWith(".dn42")).map(domain => domain.domain)
+        if (dn42Domains.length > 1) {
+            caddy.apps.tls.automation.policies.push({
+                "subjects": dn42Domains,
+                "issuers": [
+                    {
+                        "ca": "https://acme.burble.dn42/v1/dn42/acme/directory",
+                        "module": "acme"
+                    }
+                ]
+            })
+        }
+        domains.forEach(domain => {
+            caddy.apps.http.servers.srv0.routes.push({
+                "match": [
+                    {
+                        "host": [
+                            domain.domain
+                        ]
+                    }
+                ],
+                "handle": [
+                    {
+                        "handler": "subroute",
+                        "routes": [
+                            {
+                                "handle": [
+                                    {
+                                        "handler": "reverse_proxy",
+                                        "health_checks": {
+                                            "active": {
+                                                "expect_status": 2,
+                                                "interval": 5000000000,
+                                                "timeout": 60000000000
```

Comment:
Use string values instead for readability (https://caddyserver.com/docs/json/apps/http/servers/routes/handle/reverse_proxy/health_checks/active/interval/). Also, shouldn't it be reversed - the interval should be 60s and the timeout 5s?

Reply:
I used an already existing root level caddyfile and directly converted it

Reply:
swapped